Automattic/wp-calypso
# Premium Content: Add missing migrations for margin fix

#### Changes proposed in this Pull Request

* In https://github.com/Automattic/wp-calypso/pull/46579 margin issues were fixed, but the block migration functions were missed. This PR adds them.

#### Testing instructions



* Check out commit `570a7c788e02d36bccc13f220076bd856466713f`.
* Sync to your sandbox using `yarn dev --sync`
* Insert a Premium Content block on your sandboxed site and save the post/page.
* Check out this branch and resync.
* Reload the editor and confirm the migration worked (you should see this in the browser console).

Fixes invalidations in https://github.com/Automattic/wp-calypso/issues/46917
